filetime_from_git/git_wrapper.py

```diff
@@ -2,7 +2,7 @@
 """
 Wrap python git interface for compatibility with older/newer version
 """
-import itertools
+from future.moves.itertools import zip_longest
```

Comment:
Since the [`six` module is already a dependency](https://github.com/getpelican/pelican/blob/master/setup.py#L8), it's probably better to use that method. Also, using try/except blocks here might be better. Something like:

    try:
        from itertools import zip_longest
    except ImportError:
        from six.moves import zip_longest